css/main.css

```diff
@@ -145,68 +149,77 @@ kbd {
 	display: block;
 }
 
+.flexTopBar {
+	display: flex;
+	flex-direction: column;
+	width: 100%;
+	top: 0;
+	left: 0;
+	z-index: 1;/*Lets topbar render ontop of iframe*/
+	user-select: none;
+}
+
+.spectateBanner::before{
+	content: "👁️ ";
+	font-style: normal;
+}
 .spectateBanner{
 	background-color: #90FF90;
 	cursor: pointer;
-	text-align:center;
+	flex: 0 0 35%;
+	width: 99.5%; /* I think this is due to a padding issue or something? */
+	text-align: center;
+	font-style: italic;
+	background-color: lightgreen;
+	border-style: outset;
+	border-color: lightgreen;
+	font-size: 1em;
```

Comment:
I'm pretty sure this reads as 1em = 1em, since em reads font size. 
https://developer.mozilla.org/en-US/docs/Learn/CSS/Building_blocks/Values_and_units#relative_length_units

Reply:
Ultra also mentioned this. I've removed all font-size 1em instances from all the css files.